externs/shaka/player.js

```diff
@@ -664,6 +665,8 @@ shaka.extern.AdvancedDrmConfiguration;
  * @property {!Array.<string>} preferredKeySystems
  *   <i>Defaults to an empty array. </i> <br>
  *   Specifies the priorties of available DRM key systems.
+ * @property {Object.<string, string>} keySystemsMapping
+ *   A map of key system name to key system name.
```

Comment:
You should update the `drm-config.md` tutorial, specifically the "Choosing a Key System" section, to mention this config value. You can say it's a way to change the key systems in HLS. Just from the JSDoc here, what this value actually would be used for is a little bit opaque, so a mention in the tutorials would be handy.

Reply:
Done!